### ISSUE TITLE
[Accessibility] Moonlight QT UI does not work with screen readers

### ISSUE THREAD

**Issue by mrb113** [moonlight-stream/moonlight-qt#207]
Windows Narrator fails to successfully narrate the Moonlight QT UI.

**Comment by Forceflow** [comment]
This is probably relevant reading: https://doc.qt.io/qt-5/accessible.html
